tests/stylus/StylintBearTest.py

```diff
@@ -0,0 +1,233 @@
+import os
+from queue import Queue
+
+from bears.stylus.StylintBear import StylintBear
+from coalib.results.Result import Result
+from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
+from coalib.settings.Section import Section
+from coalib.testing.BearTestHelper import generate_skip_decorator
+from coalib.testing.LocalBearTestHelper import LocalBearTestHelper
+from coala_utils.ContextManagers import prepare_file
+
+test_bad_missing_colon = """
+test = @block
+    margin 0
+"""
+
+test_bad_duplicates = """
+test = @block
+    margin: 0
+
+test =
+    margin: auto
+"""
+
+test_bad_no_important = """
+.im-important
+margin 0 !important
+"""
+
+test_bad_brackets = """
+.pageContent {
+  padding: 1em
+}
+"""
+
+test_bad_semicolon = """
+.bankTitle
+  font-weight: bold;
+  padding-right: 2em
+"""
+
+test_bad_alphabetical_order = """
+.bankTitle
+  padding-right: 2em
+  font-weight: bold
+"""
+
+# Start ignoring P*, SpaceConsistencyBear
```

Comment:
Some files had mixed tabs and spaces, some had trailing whitespaces which were detected by `SpaceConsistencyBear` and `Pep8Bear`